Bottom: 34aea2338028a7b95d4f4ed946e0896de4655319
Top:    102aa014762405a776472126eaa4ab4fe12ad6be
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 22:45:13 +0100

hid/gtk: Remove dead "MovePointer" action calling code.

This was not working correctly anyway, as the action was not
properly registered. Obviously no-one missed it, so let it die.


---

diff --git a/src/hid/gtk/gui-misc.c b/src/hid/gtk/gui-misc.c
index 730db5e..79fe3b4 100644
--- a/src/hid/gtk/gui-misc.c
+++ b/src/hid/gtk/gui-misc.c
@@ -293,7 +293,7 @@ ghid_restore_cursor (void)
 static gboolean got_location;
 
   /* If user hits a key instead of the mouse button, we'll abort unless
-     |  it's one of the cursor keys.  Move the layout if a cursor key.
+     |  it's the enter key (which accepts the current crosshair location).
    */
 static gboolean
 loop_key_press_cb (GtkWidget * drawing_area, GdkEventKey * kev,
@@ -308,58 +308,8 @@ loop_key_press_cb (GtkWidget * drawing_area, GdkEventKey * kev,
   state = (GdkModifierType) (kev->state);
   mk = ghid_modifier_keys_state (&state);
 
-  /* Duplicate the cursor key actions in gui-output-events.c
-   */
   switch (ksym)
     {
-    case GDK_Up:
-      if (mk == CONTROL_PRESSED)
-	{
-	  hid_actionl ("Display", "Scroll", "8", NULL);
-	  hid_actionl ("Display", "Scroll", "0", NULL);
-	}
-      else if (mk == SHIFT_PRESSED)
-	hid_actionl ("MovePointer", "0", "-10", NULL);
-      else if (mk == NONE_PRESSED)
-	hid_actionl ("MovePointer", "0", "-1", NULL);
-      break;
-
-    case GDK_Down:
-      if (mk == CONTROL_PRESSED)
-	{
-	  hid_actionl ("Display", "Scroll", "2", NULL);
-	  hid_actionl ("Display", "Scroll", "0", NULL);
-	}
-      else if (mk == SHIFT_PRESSED)
-	hid_actionl ("MovePointer", "0", "10", NULL);
-      else if (mk == NONE_PRESSED)
-	hid_actionl ("MovePointer", "0", "1", NULL);
-      break;
-
-    case GDK_Left:
-      if (mk == CONTROL_PRESSED)
-	{
-	  hid_actionl ("Display", "Scroll", "4", NULL);
-	  hid_actionl ("Display", "Scroll", "0", NULL);
-	}
-      else if (mk == SHIFT_PRESSED)
-	hid_actionl ("MovePointer", "-10", "0", NULL);
-      else if (mk == NONE_PRESSED)
-	hid_actionl ("MovePointer", "-1", "0", NULL);
-      break;
-
-    case GDK_Right:
-      if (mk == CONTROL_PRESSED)
-	{
-	  hid_actionl ("Display", "Scroll", "6", NULL);
-	  hid_actionl ("Display", "Scroll", "0", NULL);
-	}
-      else if (mk == SHIFT_PRESSED)
-	hid_actionl ("MovePointer", "10", "0", NULL);
-      else if (mk == NONE_PRESSED)
-	hid_actionl ("MovePointer", "1", "0", NULL);
-      break;
-
     case GDK_Return:		/* Accept cursor location */
       if (g_main_loop_is_running (*loop))
 	g_main_loop_quit (*loop);
